app/components/templates/Resume.jsx

```diff
@@ -111,25 +121,49 @@ export default class Resume extends BaseTemplate {
       );
     }
 
-    return (
-      <div className="description">
-        {center}
-        {table}
-      </div>
-    );
+    return table;
   }
 
   render() {
     const data = this.getData();
 
     const expertises = this.createList(data, 'expertises');
 
+    if (this.props.data.theme && this.props.data.theme == "light") {
```

Comment:
`this.props.data.theme === "light"` should be enough of a condition :)

Reply:
Old resume does not have metadata key theme

Reply:
Yes but then it will be `undefined` so no problem with checking it. :)